backend/app/store/engine/bolt.go

```diff
@@ -16,12 +16,13 @@ import (
 )
 
 // BoltDB implements store.Interface, represents multiple sites with multiplexing to different bolt dbs. Thread safe.
-// there are 5 types of top-level buckets:
+// there are 6 types of top-level buckets:
 //  - comments for post in "posts" top-level bucket. Each url (post) makes its own bucket and each k:v pair is commentID:comment
 //  - history of all comments. They all in a single "last" bucket (per site) and key is defined by ref struct as ts+commentID
 //    value is not full comment but a reference combined from post-url+commentID
 //  - user to comment references in "users" bucket. It used to get comments for user. Key is userID and value
 //    is a nested bucket named userID with kv as ts:reference
+//  - users details in "user_details" top-level bucket. Each detail makes its own bucket and each k:v pair is userID:value
```

Comment:
generally aproach of limiting user detail value to a singe "thing" (email in this example) and support sub-buckets for each one feels odd and hard to maintain. I'd rather have a single, top-level bucket with userID as a key and struct (serilaized json for example) as the value. This way everyting can be much easier to change/extend. This is epseically valid concern for migrator support